### PR TITLE
Added few more commands

### PR DESCRIPTION
**Pull request by siva10x**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Security Best practices
+1. [https://snyk.io/blog/ten-npm-security-best-practices/](https://snyk.io/blog/ten-npm-security-best-practices/)
```
Comment (sridharrajs) [inline SECURITY.md]: Can you please expand on the list here for the quick reference?

Comment (siva10x) [reply SECURITY.md]: sure

**Comment by siva10x** [comment]
Add few security best practices to SECURITY.md file [commitid](https://github.com/sridharrajs/node-cheatsheet/pull/1/commits/ed50d484307816032ced086335ce1bf340efa4ac)

File: SECURITY.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+
+3. Use HTTPS always
+
+4. Audit the third party dependencies in your application to avoid security vulnerabilities. [Read this](https://docs.npmjs.com/auditing-package-dependencies-for-security-vulnerabilities)
+
+5. Handle the errors and hide the details from client. [Express error handling](https://expressjs.com/en/guide/error-handling.html)
+
+6. Embrace Linter security rules to identify and prevent vulnerabilities in the development phase itself. Use [tslint-config-security](https://www.npmjs.com/package/tslint-config-security), [eslint-plugin-security](https://github.com/nodesecurity/eslint-plugin-security)
+
+7. // Todo
```
Comment (sridharrajs) [inline SECURITY.md]: todo?

Comment (siva10x) [reply SECURITY.md]: I kept it as placeholder for adding future learnings. Removed it now as it is implict

**Review by sridharrajs — APPROVED** [review]
thanks for the PR @sivasankar-r

**Pull request merged**